aware-core/src/main/java/com/aware/Aware.java

```diff
@@ -2138,6 +2140,9 @@ public void onReceive(Context context, Intent intent) {
 
                         Intent aware = new Intent(context, Aware.class);
                         context.startService(aware);
+                        // Start the foreground service only if it's the client or a standalone application
```

Comment:
This check is not needed because the first line of the onReceive already makes sure that only the client and standalone execute these instructions 👍